assets/model_monitoring/components/src/model_monitor_feature_selector/run.py

```diff
@@ -33,6 +33,7 @@ def run():
         return
     elif not input_df1:
         input_df1 = input_df2
+    # TODO what if input_df1 and not input_df2?
```

Comment:
in line 28, I think we can also apply the try_read_mltable_in_spark_with_error, because input_df1 is required.

Reply:
from the current code logic, df1 is not required, it can fallback to df2. I further read the code, it's ok if df1 and not df2